webapp/src/index.tsx

```diff
@@ -40,7 +38,6 @@ root.render(
         <Routes>
           <Route path="/" element={<LandingPage />} />
           <Route path={LOGIN_OPTIONS_URL} element={<LoginPage />} />
-          <Route path={REGISTER_URL} element={<RegisterPage />} />
```

Comment:
why do you remove the register page in a PR the scope of which is limited to adding a component?

Reply:
Sorry, same as before, I tried to focus on this PR but starting from another branch, but I mess everything a little. I will be careful when merging